pkgs/development/python-modules/python-ipware/default.nix

```
@@ -0,0 +1,27 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, pythonOlder
+}:
+
+buildPythonPackage rec {
+  pname = "python-ipware";
+  version = "0.9.0";
+  format = "setuptools";
+
+  disabled = pythonOlder "3.7";
+
+  src = fetchPypi {
+    inherit pname version;
+    hash = "sha256-7oTNFsLPhi+q4ZetX4+ubHXksfQLsTNXlEpdY93Co3M=";
+  };
+
```

```suggestion
  pythonImportsCheck = [
    "ipware"
  ];

  nativeCheckInputs = [
    pytestCheckHook
  ];
```

Please enable tests like so.

---

Thanks for catching my oversight! I had to switch to using the source from GitHub directly because the unittests are not included in the PyPi tarball.